exa/__init__.py

```diff
@@ -6,18 +6,73 @@
 #########
 """
 import os
-import tempfile
+import sys
 import logging.config
+
 import yaml
+from traitlets import Unicode, default, validate
+from traitlets.config import Configurable, Application
+
+
+_app = Application()
+try:
+    _app.parse_command_line(sys.argv)
+except SystemExit:
+    pass
+_base = os.path.dirname(__file__)
+_path = os.path.join(_base, 'conf', 'config.py')
+_app.load_config_file(_path)
+
+
+class Base:
+    """This base class provides a configured
+    log property and access to configuration
+    driven application settings without
+    forcing subclasses to be run explicitly
+    in the context of an application.
+    """
+
+    @property
+    def log(self):
+        name = '.'.join([
+            self.__module__, self.__class__.__name__
+        ])
+        return logging.getLogger(name)
+
+    def __init__(self, *args, **kws):
+        kws.pop('config', None)
+        super().__init__(
```

Comment:
This implies that exa 1.0 will only support python 3. I'm good with this change.

Reply:
Right, that was initially just an oversight but I think that in 2015 it was important to keep py2 compat but in 2020 less so.